CMakeLists.txt

```diff
@@ -56,6 +56,12 @@ if(CCACHE_PROGRAM)
     set(CMAKE_CXX_COMPILER_LAUNCHER "${CCACHE_PROGRAM}")
 endif()
 
+if (USE_LLD)
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -fuse-ld=lld")
```

Comment:
Can you make USE_LLD an option so that we can set it through TAICHI_CMAKE_ARGS?

Reply:
I think you can, I did that on my end.

Reply:
Yea I did try that and it worked :P . I meant we should add it in this PR to make it more usable. 😁